package.json

```diff
@@ -13,8 +13,9 @@
     "lint:fix": "next lint --fix",
     "prettier": "prettier -w \"{src,cypress,mocks,scripts}/**/*.{ts,tsx,css,js}\"",
     "fix": "yarn lint:fix && ts-prune && yarn prettier",
-    "test": "cross-env TZ=CET DEBUG_PRINT_LIMIT=30000 jest",
+    "test": "cross-env TZ=CET DEBUG_PRINT_LIMIT=30000 jest --watchAll",
```

Comment:
is the `--watchAll` addition intended?

Reply:
oh! nope, I prefer the `--watchAll` flag to work with test. I will remove it